charts/growthbook/templates/deployment.yaml

```diff
@@ -39,6 +51,21 @@ spec:
               value: |-
                 {{- tpl $value $ | nindent 16 }}
             {{- end }}
+            - name: JWT_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "growthbook.fullname" . }}
+                  key: JWT_SECRET
+            - name: ENCRYPTION_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "growthbook.fullname" . }}
+                  key: ENCRYPTION_KEY
+            - name: MONGODB_URI
```

Comment:
I am against ENV URIs, they make debugging difficult since the host username and port are all going to be masked up. I would rather you keep non-secrets as regular env vars and only the password should be a secret. Then you can build the URI in helpers.